### DISCUSSION TITLE
Language

### DISCUSSION THREAD

**Discussion by Mednaceex** [maruohon/minihud#369] [Q&A] [answered]
When I set my game language to Russian, the mod interface also automatically "translates". This translation is mostly wrong and incomprehensible, so I'd like to revert it back. How can I leave the interface in English while keeping the game itself in another language?
Only seeems to be an issue for 1.21, the previous versions still have normal English.

**Comment by sakura-ryoko** [comment, marked as answer]
I would recommend either helping translate the mod (fixing the ru_ru.json file for us), or editing the JAR file and deleting the ru_ru.json

**Reply by Mednaceex** [reply]
It worked, thank you. I'd love to help translate the file when I have time btw
